openml-utils/src/test/java/com/feedzai/util/provider/AbstractProviderModelBaseTest.java

```diff
@@ -356,6 +396,15 @@ protected L getMachineLearningModelLoader(final MLAlgorithmEnum algorithm) {
      */
     public abstract Instance getDummyInstance();
 
+    /**
+     * Gets a dummy {@link Instance} to be classified by a {@link ClassificationMLModel} that returns a results from
```

Comment:
"that returns a results from" <-- incomplete and wrong, you mean "different results from those of..." right?